lib/config/index.js

```diff
@@ -190,9 +190,15 @@ async function getConfig() {
     config.pgPlugin.read = await getParameter('db_reader_host')
     config.pgPlugin.write = await getParameter('db_host')
     config.pgPlugin.config.port = Number(await getParameter('db_port'))
-    config.pgPlugin.config.ssl = /true/i.test(await getParameter('db_ssl'))
     config.pgPlugin.config.database = await getParameter('db_database')
 
+    if (/true/i.test(await getParameter('db_ssl'))) {
+      config.pgPlugin.config.ssl = {
+        ca: [rdsSecret.ca],
+        rejectUnauthorized: true
+      }
+    }
+
     config.routes.callback = /true/i.test(await getParameter('enable_callback'))
     config.routes.checkIn = /true/i.test(await getParameter('enable_check_in'))
     config.routes.metrics = /true/i.test(await getParameter('enable_metrics'))
```

Comment:
FWIW... using the regex patterns like this is not that efficient from a memory or performance point of view. There are definitely some improvements that could be made in here but certainly a low priority